### PR TITLE
Feature/lgcs 414

### PR DESCRIPTION
**Pull request by joel-stobart**

### CODE REVIEW

**Review by adamgparsons — APPROVED** [review]

File: CyberHealth/assessment/views.py [hunk 1]
```diff
@@ -8,14 +7,24 @@
...
+    for question in questions:
+        if question.answer_set.all().first() is None:
+            question.chosen_answer = "None"
+            question.answer_colour = "blue"
+        else:
+            question.chosen_answer = question.answer_set.all().first().choice.choice_text
+            if question.chosen_answer == "yes":
+                question.answer_colour = "green"
+            else:
+                question.answer_colour = "red"
```
Comment (kingdomlevel) [inline CyberHealth/assessment/views.py:25]: I see what you mean about doing this business logic here + passing to template!

This looks good though. Is working as expected for first answer only (which I presume is intended - e.g. once I have answered "yes" to a question, if i change to no, the `assessment_start_page` doesn't update to reflect most recent answer).

**Review by kingdomlevel — COMMENTED** [review]
Assuming the comment above is as intended, looks good to me!

File: CyberHealth/assessment/views.py (hunk 1 above)
Comment (joel-stobart) [reply CyberHealth/assessment/views.py]: I was wrong on this one - and therefore have made a change to reflect that. It should be the latest answer `last` , not the oldest answer `first`.

Great spot

Comment (kingdomlevel) [reply CyberHealth/assessment/views.py]: This is working for me now, nice work!

**Review by kingdomlevel — APPROVED** [review]
Looks good, tests passing, nice work! 🥳

**Pull request merged**